client/src/components/OnePage/Footer/Footer.tsx

```diff
@@ -1,53 +1,132 @@
-  [theme.breakpoints.down('sm')]: {
+  borderRadius: '20px',
+  display: 'flex',
+  padding: '20px',
+  [theme.breakpoints.down('md')]: {
     flexDirection: 'column',
-    alignItems: 'center',
+    height: 'auto',
+    padding: '15px',
   },
 }));
 
-const FooterContent = styled(Box)(({ theme }) => ({
+const SloganBox = styled(Box)(({ theme }) => ({
+  width: '50%',
+  fontSize: '26px',
+  fontWeight: 'bold',
+  [theme.breakpoints.down('md')]: {
+    marginTop: '0',
+    width: 'auto',
+    fontSize: '18px',
+    fontWeight: 350,
+  },
+}));
+
+const GroupsBox = styled(Box)(({ theme }) => ({
+  alignItems: 'flex-start',
   display: 'flex',
-  color: 'white',
-  flexDirection: 'column',
-  justifyContent: 'center',
-  alignItems: 'center',
-  textAlign: 'center',
-  maxWidth: 895,
+  justifyContent: 'flex-start',
   [theme.breakpoints.down('md')]: {
-    marginTop: '150px',
+    width: 'auto',
   },
 }));
 
-const FooterEndLine = styled(Box)(() => ({
+const CustomBoxGroup = styled(Box)(({ theme }) => ({
+  width: '190px',
+  marginRight: '40px',
+  [theme.breakpoints.down('md')]: {
+    marginRight: '24px',
+    width: 'auto',
+    fontSize: '14px',
+    fontWeight: 500,
+    marginBottom: '10px',
+  },
+}));
+
+const CopyrightContainer = styled(Container)(({ theme }) => ({
   alignItems: 'center',
-  textAlign: 'center',
+  display: 'flex',
+  justifyContent: 'space-between',
+  marginBottom: '20px',
+  marginTop: '20px',
+  color: '#FF6700',
+  fontWeight: 'bold',
+  [theme.breakpoints.down('md')]: {
+    alignItems: 'flex-start',
+    flexDirection: 'column',
+    fontSize: '12px',
+    fontWeight: 350,
+    justifyContent: 'flex-start',
+    marginBottom: '35px',
+    marginTop: '20px',
+  },
+}));
+
+const CustomTermsBox = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  [theme.breakpoints.down('md')]: {
+    fontSize: '0.75rem',
+  },
+}));
+
+const CustomTermsTypography = styled('p')(({ theme }) => ({
+  marginLeft: '20px',
+  [theme.breakpoints.down('md')]: {
+    margin: '4px 8px 0 0',
+  },
 }));
 
 const Footer = () => {
   return (
-    <FooterContainer
-      maxWidth={false}
-      disableGutters
-      className="container full-width"
+    <Container
+      className="dark-container"
+      sx={{ paddingTop: '120px', paddingBottom: '10px', position: 'relative' }}
     >
-      <FooterContent className="footer-content">
-        <FooterEndLine className="footer-end-items">
-          <Typography>Copyright © 2023 Chattie</Typography>
-          <Typography>Created by Kinga Kuś & Adam Kostuch</Typography>
-        </FooterEndLine>
-      </FooterContent>
-    </FooterContainer>
+      <CustomContainer className="footerContainer">
+        <CustomBoxItems className="footer-items">
+          <SloganBox className="slogan">
+            <h3>
+              Hello. We are CHATTIE,
+              <br />
+              connecting people with others
+              <br />
+              and with robots.
+            </h3>
```

Comment:
Do we need to use break lines? Or can't we just leave it without it? It would be for sure more flexible. If you want to layout it this way, you may want to specify the width of the component: 
```suggestion
            <Typoghraphy variant="h3" width="x%">
              Hello. We are CHATTIE,
              connecting people with others
              and with robots.
            </Typoghraphy>
```

Reply:
At this moment I want to leave the br